cypress/e2e/bulk-edit/csv/bulk-edit-location.cy.js

```diff
@@ -0,0 +1,138 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { Locations } from '../../../support/fragments/settings/tenant/location-setup';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import FileManager from '../../../support/utils/fileManager';
+import ExportFile from '../../../support/fragments/data-export/exportFile';
+import DateTools from '../../../support/utils/dateTools';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+
+describe('bulk-edit', () => {
```

Comment:
we need 2 describe blocks here, like this
describe('bulk-edit', () => {
  describe('in-app approach', () => {